README.md

```diff
@@ -1,42 +1,14 @@
-# Foxglove Studio WebSocket protocol libraries
+# Foxglove WebSocket Protocol
 
-This repository provides a protocol specification and reference implementations enabling [Foxglove Studio](https://github.com/foxglove/studio) to ingest arbitrary “live” streamed data.
+This repository provides a [protocol specification](docs/spec.md) and reference implementations enabling [Foxglove Studio](https://github.com/foxglove/studio) to ingest arbitrary “live” streamed data.
 
-The protocol is encoding-agnostic, i.e. it can support Protobuf messages, JSON messages, etc. (as long as the desired encoding is supported by both client and server).
+The protocol is encoding-agnostic, and supports any data format as long as the desired encoding is supported by both client and server.
 
-The following implementations are provided in this repository and as installable packages:
+Implementations are available in the following languages:
 
-| Language              | Includes         | Package name                     | Version                                                                                                                      |
-| --------------------- | ---------------- | -------------------------------- | ---------------------------------------------------------------------------------------------------------------------------- |
-| Python                | server           | `foxglove-websocket`             | [![](https://shields.io/pypi/v/foxglove-websocket)](https://pypi.org/project/foxglove-websocket/)                            |
-| JavaScript/TypeScript | server + client  | `@foxglove/ws-protocol`          | [![](https://shields.io/npm/v/@foxglove/ws-protocol)](https://www.npmjs.com/package/@foxglove/ws-protocol)                   |
-| JavaScript/TypeScript | examples         | `@foxglove/ws-protocol-examples` | [![](https://shields.io/npm/v/@foxglove/ws-protocol-examples)](https://www.npmjs.com/package/@foxglove/ws-protocol-examples) |
-| C++                   | _in development_ |
-
-## Documentation
-
-- [Protocol specification](docs/spec.md)
-- [Python package README](python)
-- [JavaScript/TypeScript package README](typescript/ws-protocol)
-
-## Development
-
-### Virtualenv usage
-
-```
-pipenv install --dev
-pipenv shell
-```
-
-### Run example servers
-
-```
-python -m foxglove_websocket.examples.simple_server
-npx @foxglove/ws-protocol-examples sysmon
-```
-
-### Run example client
-
-```
-npx @foxglove/ws-protocol-examples simple-client localhost:8765
```

Comment:
Should this be `ws://localhost:8765`?

Reply:
Nope, the ws:// is added in the code before constructing the WebSocket client